src/test/rustdoc/issue-83832.rs

```diff
@@ -0,0 +1,21 @@
+#![crate_name = "foo"]
+
+pub mod io {
+    #[deprecated(since = "0.1.8", note = "Use bar() instead")]
```

Comment:
Can you also add a test for unstable items?

Reply:
Done.